Can't find resource for bundle dev.jbang.cli.JBang$ConfigurationResourceBundle, key preview

Get this error: 

```
[jbang] [0:417] jbang version 0.106.1.2
[jbang] [0:438] [ERROR] Can't find resource for bundle dev.jbang.cli.JBang$ConfigurationResourceBundle, key preview
java.util.MissingResourceException: Can't find resource for bundle dev.jbang.cli.JBang$ConfigurationResourceBundle, key preview
	at java.base/java.util.ResourceBundle.getObject(ResourceBundle.java:564)
	at java.base/java.util.ResourceBundle.getString(ResourceBundle.java:521)
	at picocli.CommandLine$Model$Messages.addAllWithPrefix(CommandLine.java:11402)
	at picocli.CommandLine$Model$Messages.getStringArray(CommandLine.java:11398)
	at picocli.CommandLine$Model$ArgSpec.description(CommandLine.java:8719)
	at picocli.CommandLine$Help$DefaultOptionRenderer.renderDescriptionLines(CommandLine.java:16223)
	at picocli.CommandLine$Help$DefaultOptionRenderer.render(CommandLine.java:16196)
	at picocli.CommandLine$Help.calcLongOptionColumnWidth(CommandLine.java:16022)
	at picocli.CommandLine$Help.createDefaultLayout(CommandLine.java:16000)
	at picocli.CommandLine$Help.createDefaultLayout(CommandLine.java:15991)
	at picocli.CommandLine$Help.parameterList(CommandLine.java:15655)
	at picocli.CommandLine$Help.parameterList(CommandLine.java:15645)
	at picocli.CommandLine$Model$UsageMessageSpec$10.render(CommandLine.java:7856)
	at picocli.CommandLine.usage(CommandLine.java:2682)
	at picocli.CommandLine.usage(CommandLine.java:2654)
	at picocli.CommandLine.usage(CommandLine.java:2610)
	at dev.jbang.cli.JBang.doCall(JBang.java:98)
	at dev.jbang.cli.BaseCommand.call(BaseCommand.java:145)
	at dev.jbang.cli.BaseCommand.call(BaseCommand.java:21)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
	at dev.jbang.cli.JBang$3.handle(JBang.java:147)
	at dev.jbang.cli.JBang$3.handle(JBang.java:142)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at dev.jbang.Main.main(Main.java:14)
[jbang] [0:439] If you believe this a bug in jbang, open an issue at https://github.com/jbangdev/jbang/issues
```

I have this in config:
```
 jbang config list
format = text
init.template = hello
jdkproviders = current,default,javahome,path,jbang
preview = true
run.debug = 4004
run.jfr = filename={baseName}.jfr
wrapper.install.dir = .
```

preview might not be known but shouldn't error here - at most warn ?

--- a/src/main/java/dev/jbang/cli/Edit.java
+++ b/src/main/java/dev/jbang/cli/Edit.java
@@ -48,7 +48,7 @@ public class Edit extends BaseCommand {
 	public boolean live;
 
 	@CommandLine.Option(names = {
-			"--open" }, arity = "0..1", defaultValue = "${JBANG_EDITOR:-${jbang.edit.open:-}}", fallbackValue = "${JBANG_EDITOR:-${jbang.edit.open:-}}", description = "Opens editor/IDE on the temporary project.", preprocessor = StrictParameterPreprocessor.class)
+			"--open" }, arity = "0..1", defaultValue = "${JBANG_EDITOR:-${default.edit.open:-}}", fallbackValue = "${JBANG_EDITOR:-${default.edit.open:-}}", description = "Opens editor/IDE on the temporary project.", preprocessor = StrictParameterPreprocessor.class)
 	public Optional<String> editor;
 
 	@CommandLine.Option(names = { "--no-open" })

--- a/src/main/java/dev/jbang/cli/JBang.java
+++ b/src/main/java/dev/jbang/cli/JBang.java
@@ -22,6 +22,7 @@ import java.util.Objects;
 import java.util.ResourceBundle;
 import java.util.Set;
 import java.util.concurrent.Future;
+import java.util.stream.Collectors;
 
 import dev.jbang.Configuration;
 import dev.jbang.util.Util;
@@ -213,10 +214,13 @@ public class JBang extends BaseCommand {
 	}
 
 	static class ConfigurationResourceBundle extends ResourceBundle {
+
+		private static final String PREFIX = "default.";
+
 		@Override
 		protected Object handleGetObject(String propkey) {
-			if (propkey.startsWith("jbang.")) {
-				String key = propkey.substring(6);
+			if (propkey.startsWith(PREFIX)) {
+				String key = propkey.substring(PREFIX.length());
 				return Configuration.instance().get(key);
 			} else {
 				return null;
@@ -225,7 +229,12 @@ public class JBang extends BaseCommand {
 
 		@Override
 		public Enumeration<String> getKeys() {
-			return Collections.enumeration(Configuration.instance().keySet());
+			return Collections.enumeration(Configuration.instance()
+														.flatten()
+														.keySet()
+														.stream()
+														.map(k -> "default." + k)
+														.collect(Collectors.toSet()));
 		}
 	}
 

--- a/src/main/java/dev/jbang/cli/RunMixin.java
+++ b/src/main/java/dev/jbang/cli/RunMixin.java
@@ -12,11 +12,11 @@ public class RunMixin {
 	public List<String> javaRuntimeOptions;
 
 	@CommandLine.Option(names = {
-			"--jfr" }, fallbackValue = "${jbang.run.jfr}", parameterConsumer = Run.KeyValueFallbackConsumer.class, arity = "0..1", description = "Launch with Java Flight Recorder enabled.")
+			"--jfr" }, fallbackValue = "${default.run.jfr}", parameterConsumer = Run.KeyValueFallbackConsumer.class, arity = "0..1", description = "Launch with Java Flight Recorder enabled.")
 	public String flightRecorderString;
 
 	@CommandLine.Option(names = { "-d",
-			"--debug" }, fallbackValue = "${jbang.run.debug}", parameterConsumer = Run.DebugFallbackConsumer.class, arity = "0..1", description = "Launch with java debug enabled on specified port (default: ${FALLBACK-VALUE}) ")
+			"--debug" }, fallbackValue = "${default.run.debug}", parameterConsumer = Run.DebugFallbackConsumer.class, arity = "0..1", description = "Launch with java debug enabled on specified port (default: ${FALLBACK-VALUE}) ")
 	public String debugString;
 
 	// should take arguments for package/classes when picocli fixes its flag

--- a/src/main/resources/jbang.properties
+++ b/src/main/resources/jbang.properties
@@ -1,6 +1,6 @@
+format=text
 init.template=hello
+jdkproviders=current,default,javahome,path,jbang
 run.debug=4004
 run.jfr=filename={baseName}.jfr
 wrapper.install.dir=.
-format=text
-jdkproviders=current,default,javahome,path,jbang

